saleor/payment/utils.py

```diff
@@ -33,26 +33,33 @@ def create_payment_information(
     customer_id: str = None,
     store_source: bool = False,
     additional_data: Optional[dict] = None,
+    partial: bool = False,
 ) -> PaymentData:
     """Extract order information along with payment details.
 
     Returns information required to process payment and additional
     billing/shipping addresses for optional fraud-prevention mechanisms.
     """
     checkout = payment.checkout
+    checkout_token = None
     if checkout:
         billing = checkout.billing_address
         shipping = checkout.shipping_address
         email = checkout.get_customer_email()
         user_id = checkout.user_id
+        checkout_token = str(checkout.token)
     elif payment.order:
         billing = payment.order.billing_address
         shipping = payment.order.shipping_address
         email = payment.order.user_email
         user_id = payment.order.user_id
+        checkout_token = str(payment.order.checkout_token)
```

Comment:
Do we need to apply `str` here? It seems that `checkout_token` is a CharField.

Reply:
No, changed.